types/primer__octicons/index.d.ts

```diff
@@ -118,10 +119,12 @@ declare namespace octicons {
         | 'broadcast'
         | 'browser'
         | 'bug'
+		| 'cache'
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/primer__octicons/index.d.ts;
  Line: 122;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.